.github/workflows/python-build-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8, 3.9, '3.10', 3.11]
```

Comment:
Why ' around the 3.10 only?

Reply:
I could do it around all of them if we want.  but to answer your specific question, it is because 3.10 is treated as a number and the 0 is chopped off so it tried testing using 3.1 rather than 3.10 if you don't add the quotes.  The rest resolve fine as is.  I am definitely fine putting a quote around all of them though just for consistency

Reply:
Right ok sounds good

Reply:
I've added quotes to all of these in the updated version.  Should I resolve the conversation or should you? I have done it both ways in different repos